core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -732,6 +740,9 @@ class RealityTreeReference extends RealityModelTileTree.Reference {
         drawArgs.location.origin.z -= elevationBias;
     }
 
+    if (undefined !== drawArgs && undefined !== this._clips)
+      drawArgs.clipVolume = IModelApp.renderSystem.createClipVolume(this._clips);
```

Comment:
Preferable to create the clip volume once and store it.

Reply:
Done, see 06c9c7d.